paragon/_account.scss

```
@@ -0,0 +1,324 @@
+.page__account-settings {
```

Please consider moving it into the Variables Scss file.

```suggestion
:root {
    --fx-concrete-color: $concrete;
}


.page__account-settings {
```